quinn-h3/src/qpack/prefix_string/decode.rs

```diff
@@ -78,6 +75,29 @@ impl HuffmanDecoder {
     }
 }
 
+// Read `len` bits from the `src` slice at the specified position
+//
+// Never read more than 8 bits at a time. `bit_offset` may be larger than 8.
```

Comment:
nit: these might as well be doc comments

Reply:
Fixed.